### PR TITLE
Yahhh

### PR DESCRIPTION
**Pull request by Timlllllli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
-# The ultimate travel guide
- - [AFRICA](./africa/README.md)
+# Worldwise guide
+
```
Comment (dpshelio) [inline README.md:2]: These changes are not needed. Or at least requires an explanation to explain why a change of name is suggested :wink:

File: README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
  - [ASIA](./asia/README.md)
  - [EUROPE](./europe/README.md)
  - [NORTH_AMERICA](./north_america/README.md)
  - [OCEANIA](./oceania/README.md)
  - [SOUTH_AMERICA](./south_america/README.md)
+ - [Africa](./africa/README.md)
+
+ - [Shanxi](https://github.com/Timlllllli/Travel-guide/blob/yahhh/asia/china/Shanxi.md)
```
Comment (dpshelio) [inline README.md:10]: This link is not required. Its content is already under the contribution to this repository.

File: asia/china/Shanxi.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# Places of interest
+ - GuanGong temple
+ - More info at 
+   [wikivoyage]
+   (https://en.wikivoyage.org/wiki/Yuncheng) and [wikipedia]
+   (https://en.wikipedia.org/wiki/Yuncheng)
+
+# Add the internal links
+- [README](./README.md)
```
Comment (dpshelio) [inline asia/china/Shanxi.md:12]: This link points to China's readme, so it's better to add a more descriptive name to this link :wink: 
```suggestion
- [China](./README.md)
```

**Review by dpshelio — CHANGES_REQUESTED** [review]
Great contribution! Thanks.

A couple of things to keep an eye for:
- commit messages need to be descriptive
- Pull request title should be meaningful, something like the location you are adding is preferred.

Please, follow the suggestions underr this review so that we can merge your contribution!

**Review by zwei21 — COMMENTED** [review]
A cluster  of dir is required, for example, the internal link should be held for the following structures:
Link1:Asia/README->China/README
Link2:China/README->Shanxi.md
Other descriptions and internal links is not required in the issue.

**Pull request closed without merging**